packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.doc.tsx

```diff
@@ -24,6 +25,11 @@ export const TeachingBubblePageProps: IDocPageProps = {
       code: TeachingBubbleBasicExampleCode,
       view: <TeachingBubbleBasicExample />
     },
+    {
+      title: 'TeachingBubble Wide',
```

Comment:
Nit: To match this with the "TeachingBubble wide with Illustration" below, maybe we can either make this "Wide" (and "Condensed" in the next example) lowercase, or the "wide" in "TeachingBubble wide with Illustration" uppercase.